src/Compilers/CSharp/Portable/Symbols/Source/ParameterHelpers.cs

```diff
@@ -524,6 +530,10 @@ private static void EnsureNullableAttributeExists(CSharpCompilation compilation,
                         {
                             addERR_BadParameterModifiers(diagnostics, modifier, SyntaxKind.OutKeyword);
                         }
+                        else if (seenReadonly)
+                        {
+                            addERR_BadParameterModifiers(diagnostics, modifier, SyntaxKind.ReadOnlyKeyword);
```

Comment:
>addERR_BadParameterModifiers(diagnostics, modifier, SyntaxKind.ReadOnlyKeyword); [](http://example.com/codeflow?start=28&length=80)

Is this code path reachable?

Reply:
Yes, in `void M(readonly params int[] p)`.

Reply:
> Yes, in ```void M(readonly params int[] p)```.

I might be missing something, but in ```ReadonlyWithParams``` unit-test I do not see this error reported.

Reply:
In fact, I do not see ERR_BadParameterModifiers reported with "'readonly'" at the end in the entire file.

Reply:
Yes, sorry, posted the reply by mistake.